### ISSUE TITLE
request: make main menu bar floatable

### ISSUE THREAD

**Issue by pixelartgirl** [mapeditor/tiled#3709]
I'm Pixel Art Girl and I come from Pixel Land, where all people have autism!
I was send to earth to make Tiled more user friendly

**Is your feature request related to a problem? Please describe.**
make main menu bar floatable

**Describe the solution you'd like**
We can rearrange everything on the screen, except the main menu bar.
I would like it to be floatable so we can move it to the anywhere of the screen
So basically make it a docker, just like all the other windows.

**Comment by pixelartgirl** [comment]
PUT THIS SHIT UP YOUR FUCKING ASS
